v0/destinations/google_adwords_offline_conversions/transform.js

```diff
@@ -0,0 +1,138 @@
+const { EventType } = require("../../../constants");
+const {
+  CustomError,
+  getHashFromArrayWithDuplicate,
+  constructPayload,
+  removeHyphens,
+  getValueFromMessage,
+  defaultRequestConfig,
+  getErrorRespEvents,
+  getSuccessRespEvents
+} = require("../../util");
+const {
+  trackClickConversionsMapping,
+  CLICK_CONVERSION,
+  trackCallConversionsMapping,
+  CALL_CONVERSION
+} = require("./config");
+const { validateDestinationConfig, getAccessToken } = require("./utils");
+
+const getConversions = (message, metadata, { Config }, conversionType) => {
+  let payload;
+  let endpoint;
+  const filteredCustomerId = removeHyphens(Config.customerId);
+  if (conversionType === "click") {
+    payload = constructPayload(message, trackClickConversionsMapping);
+    endpoint = CLICK_CONVERSION.replace(":customerId", filteredCustomerId);
+  } else {
+    payload = constructPayload(message, trackCallConversionsMapping);
+    endpoint = CALL_CONVERSION.replace(":customerId", filteredCustomerId);
+  }
+
+  payload.partialFailure = true;
+
+  const response = defaultRequestConfig();
+  response.endpoint = endpoint;
+  response.params = { event: message.event, customerId: filteredCustomerId };
+  response.body.JSON = payload;
+  response.headers = {
+    Authorization: `Bearer ${getAccessToken(metadata)}`,
+    "Content-Type": "application/json",
+    "developer-token": getValueFromMessage(metadata, "secret.developer_token")
```

Comment:
`getValueFromMessage` shouldn't this be `get` 
`getValueFromMessage` us mainly used for message property extraction

Reply:
@utsabc Good catch. `get` is a library used to get values from a key path inside an object. This is a use-case for `get`